EasyClangComplete.sublime-settings

```diff
@@ -1,23 +1,26 @@
 {
-	// Where should clang search for headers
-	// These will be appended to build with `-I` flag each
-	// "$project_base_path" will be replaced with opened project path
-	// "$project_name" will be replaced with opened project name
-	// "$clang_version" will be replaced with numerical version str, e.g. 3.8.0
-	"include_dirs" : [
-		"/usr/include",
-		"$project_base_path/src",
-		"/workspace/$project_name/src",
-		"/usr/lib/clang/$clang_version/include" // example for Ubuntu 16.04
-		],
-
-	// set the default c standard library flag.
-	"std_flag_c" : "-std=c11",
-
-	// set the default c++ standard library flag.
-	"std_flag_cpp" : "-std=c++11",
-
-	// search for .clang_complete file up the tree. Use includes from it if found
+	// Specify common flags that will be passed to clang for EVERY build.
+	// You can use wilcards:
+	// 	- $project_base_path -> will be replaced with the project path being built
+	// 	- $project_name -> replaced by name of the project being built
+	// 	- $clang_version -> replaced by the numeric version of clang in use
+	"common_flags" : [
+		// some example includes
+		"-isystem /usr/include",
+		"-I$project_base_path/src",
+		"-I/workspace/$project_name/src",
```

Comment:
They are there to give an example what one can use without forcing the user to read the docs. That is only valuable if it doesn't break anything. It doesn't for clang. If it does for `clang-cl`, one can of course remove them. 

As for usefulness: nearly all my projects have src folder, so it makes sense as a default for me. Also I use catkin build, which puts all projects into a workspace. Although that probably has to be a relative path.